inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/convolution_gpu_b_fs_yx_fsv4_int8.cl

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "include/common.cl"
+#include "include/data_types.cl"
+#include "include/fetch.cl"
+#include "include/imad.cl"
+
+#if QUANTIZATION_TERM
+#    define ACCUMULATOR_TYPE int
+#    define TO_ACCUMULATOR_TYPE(x) convert_int(x)
+#    define ACTIVATION_TYPE float
+#    define TO_ACTIVATION_TYPE(x) convert_float(x)
+#else
+#    define ACCUMULATOR_TYPE INPUT0_TYPE
+#    define TO_ACCUMULATOR_TYPE(x) TO_INPUT0_TYPE(x)
+#    define ACTIVATION_TYPE INPUT0_TYPE
+#    define TO_ACTIVATION_TYPE(x) TO_INPUT0_TYPE(x)
+#endif
+
+__attribute__((intel_reqd_sub_group_size(SUB_GROUP_SIZE)))
+__attribute__((reqd_work_group_size(1, 1, SUB_GROUP_SIZE)))
+KERNEL(convolution_gpu_b_fs_yx_fsv4_int8)(
+    const __global float* input,
+    __global OUTPUT_TYPE* output,
+    const __global FILTER_TYPE* weights,
+#if BIAS_TERM
+    const __global BIAS_TYPE* bias,
+#endif
+#if HAS_FUSED_OPS_DECLS
+    FUSED_OPS_DECLS,
+#endif
+    uint split_idx)
+{
+#define AS_TYPE_N_(type, n, x) as_##type##n(x)
+#define AS_TYPE_N(type, n, x) AS_TYPE_N_(type, n, x)
+#define AS_INPUT0_TYPE_4(x) AS_TYPE_N(INPUT0_TYPE, 4, x)
+#define AS_FILTER_TYPE_4(x) AS_TYPE_N(FILTER_TYPE, 4, x)
+
+    const uint idx = 2 * ((uint)get_global_id(0) * 16 + (uint)get_global_id(2));
+    const uint idy = (uint)get_global_id(1);
+    uint filter_idx = 0;
+    uint output_idx = 0;
+    uint input_idx = 0;
+    int batch = 0;
+#if FILTER_OFM_NUM > 8
+#define FILTER_OFM_MAX 8
+#else
+#define FILTER_OFM_MAX FILTER_OFM_NUM
+#endif
+        __attribute__((opencl_unroll_hint(1)))
+        for (int iter = 0; iter < FILTER_OFM_NUM / FILTER_OFM_MAX + (FILTER_OFM_NUM % FILTER_OFM_MAX != 0); iter++) {
+            int out1[FILTER_OFM_MAX] = { 0 };
+            int out2[FILTER_OFM_MAX] = { 0 };
+            filter_idx = FILTER_OFM_MAX * iter * 4;
+
+            __attribute__((opencl_unroll_hint((FILTER_IFM_NUM + 3) / 4)))
+                for (int ifm = 0; ifm < (FILTER_IFM_NUM+3)/4; ifm++) {
```

Comment:
Please fix code style in the kernel here and below
e.g. `(FILTER_IFM_NUM+3)/4` ->  `(FILTER_IFM_NUM + 3) / 4`

Reply:
And fix alignment as well.

Reply:
Done